README.md

```diff
@@ -37,17 +37,57 @@ The modes change next parameters in the components (shown in the figure), with s
 
 ![pilot_overview](resources/pilot-urjc.png)
 
-## Launching pilot-urjc demo
-This pilot has been tested on different platforms. Above we show how to run the demo in each one.
-First of all, we have to download the dependencies packages. We will use **vcs-tool**
+## Build pilot-urjc demo
+
+  We will use **vcs-tool** to get the dependencies and packages, we have to create several ws.
+
+### Navigation2 dependencies
+  
+  Fetch, build and install navigation2 dependencies:
+
+  ```console
+    mkdir -p ~/ros2_nav_dependencies_ws/src
+    cd ~/ros2_nav_dependencies_ws
+    wget https://raw.githubusercontent.com/MROS-RobMoSys-ITP/Pilot-URJC/mario-tests/nav2_dependencies.repos
+    vcs import src < nav2_dependencies.repos
+    rosdep install -y -r -q --from-paths src --ignore-src --rosdistro eloquent
+    colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+
   ```
-    cd [ros2_ws]/src
-    git clone https://github.com/MROS-RobMoSys-ITP/Pilot-URJC.git
-    vcs import < Pilot-URJC/dependencies.repos
-    cd ..
-    rosdep install --from-paths src --ignore-src -r -y
+
+### Navigation2 ([MROS-RobMoSys-ITP](https://github.com/MROS-RobMoSys-ITP/mros_navigation2) fork)
+
+  Fetch, build and install navigation2 stack:
+
+  ```console
+    source ~/ros2_nav_dependencies_ws/install/setup.bash
+    mkdir -p ~/navigation2_ws/src
+    cd ~/navigation2_ws
+    wget https://raw.githubusercontent.com/MROS-RobMoSys-ITP/Pilot-URJC/mario-tests/dependencies.repos
+    vcs import src < dependencies.repos
+    rosdep install -y -r -q --from-paths src --ignore-src --rosdistro eloquent
+    colcon build --symlink-install
+  ```
+
+### Turtlebot3
+
+  Turtlebot3 is one of tests platforms. Ignore this if you are not using it.
+  Fetch, build and install turtlebot3 packages:
+
+  ```console
+    source ~/navigation2_ws/install/setup.bash
+    mkdir -p ~/turtlebot3_ws/src
```

Comment:
IMHO, we must have one WS with all necessary packages, instead of three WS.

Reply:
We need two separate WS to compile the BT and the Navigation, if we use the binaries, that will not be the case.  The third WS was for TB3, but it can of course be on the same WS as the Nav2.